source/YugiAcademy/YuGiOh/cards/spell_card.py

```diff
@@ -0,0 +1,25 @@
+from django.contrib.contenttypes.fields import GenericRelation
+from django.db import models
+
+from .card import Card
+from YuGiOh.booster_packs import BoosterPackCard
+from assertions import enforce_not_blank
+
+
+class SpellCard(Card):
+    type_description = 'spell card'
+    type = models.CharField(max_length=20)
+    set = GenericRelation(to=BoosterPackCard, related_query_name='spell_card')
+
+    @classmethod
+    def named(cls, name: str, type: str, description: str):
+        enforce_not_blank(name, "Name")
+        enforce_not_blank(type, "Type")
+        enforce_not_blank(description, "Description")
+
+        return cls(name=name, type=type, description=description)
+
+    def synchronize_with(self, spell_card):
```

Comment:
I think that synchronize it's not representative. Upgrade or replace it's better. Or something like that. Synchronize seems to be a Synchronization with a future link between objects.

Reply:
Already explained!